src/lib/components/MapActions/map-actions.css

```diff
@@ -1,10 +1,10 @@
-.FlowDiagramExplorer .MapActions {
+.FlowDiagramExplorer__MapActions {
     position: absolute;
     right: 0px;
     bottom: 0px;
 }
 
-.FlowDiagramExplorer .MapActions > * {
+button.FlowDiagramExplorer__FloatingActionButton {
```

Comment:
Reconsidering this, it might be nicer to include `MapActions` in the name to make it clear that only `FAB`s in the `MapActions` component are styled like this. Suggestion:
`button.FlowDiagramExplorer__MapActions__FAB`  (so that it's not getting too long).

Reply:
The button component is named `Fab`, thereby renamed to `button.FlowDiagramExplorer__MapActions__Fab`